bin/collator/Cargo.toml

```diff
@@ -114,6 +114,7 @@ frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch =
 frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", optional = true }
 pallet-xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.16", optional = true }
 xcm-builder = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.16", optional = true }
+pallet-society = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", optional = true }
```

Comment:
Looks like it’s useless

Reply:
Yeah, not sure why exactly this happens but it seems that one of the dependencies includes pallet-society and this messes up the build if we don't enable `runtime-benchmarks` for it.

Reply:
`polkadot-cli` includes some `performance-tests` which include `kusama-runtime` which includes `pallet-society`.

Reply:
@akru I've cleaned it up - please approve.